core_gems.rb

```diff
@@ -12,3 +12,4 @@
 download "tzinfo-data", "1.2016.5"
 fetch "google-protobuf", "3.6.1"
 fetch "grpc", "1.8.3"
+download "fluentd", "1.2.5"
```

Comment:
Does it not come from [here](https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/config/software/fluentd.rb#L3)?